style.css

```diff
@@ -440,3 +453,476 @@ li {
 .form-body {
   background-color: #fff;
 }
+
+/* Desktop */
+
+@media only screen and (min-width: 900px) {
+  .nav-mobile {
+    display: none;
+  }
+
+  .mobile-image {
+    display: none;
+  }
+
+  .desktop-image,
+  .rails {
+    display: inherit;
+  }
+
+  * {
+    font-family: Poppins, sans-serif;
+    padding: 0;
+    margin: 0;
+  }
+
+  .main-block {
+    background-color: #e5e5e5;
+  }
+
+  .head-menu {
+    background-color: #fff;
+  }
+
+  .nav-desktop {
+    display: flex;
+    align-items: center;
+    padding: 3vh 10vw;
+    justify-content: space-between;
+    box-sizing: border-box;
+    width: 100vw;
+    background-color: #fff;
+  }
+
+  .logo-desktop {
+    color: #6070ff;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 1.5vh;
+  }
+
+  .nav-element-desktop {
+    display: flex;
+    gap: 3vw;
+    align-items: center;
+    text-align: center;
+  }
+
+  .elements {
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.5vh;
+  }
+
+  .home-page {
+    background-image: url("./Images/home-page.svg");
+    background-repeat: no-repeat;
+    background-size: 100vw;
+    background-color: #fff;
+    height: 75vh;
+    border-bottom-left-radius: 10vh;
+  }
+
+  .home-page-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .height {
+    margin-top: 18vh;
+  }
+
+  .intro {
+    width: 35vw;
+    font-size: 5vh;
+    line-height: 7vh;
+    font-weight: 700;
+    color: #091e42;
+    padding-left: 15vw;
+  }
+
+  .intro-description {
+    width: 75vw;
+    font-size: 2vh;
+    font-weight: 400;
+    color: #344563;
+    margin-top: 1vh;
+    margin-bottom: 1vh;
+    padding-left: 15vw;
+  }
+
+  .bottom-connect {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 1.5vh;
+    color: #7f8cff;
+    margin-top: 1vh;
+    padding: 0;
+    width: 10vw;
+  }
+
+  .icon-container {
+    display: flex;
+    list-style: none;
+    color: #505f79;
+    margin: 0;
+    padding-left: 17vw;
+  }
+
+  .icon-img {
+    width: 2vw;
+    height: 2vh;
+    padding-right: inherit;
+  }
```

Comment:
- [ ] I suggest you remove these hard-coded widths based on screen size as they make your images stretch on larger screens, as shown below.

![](https://i.imgur.com/tpsnPsM.png)

Reply:
Thanks for the remark. I will fix these.